x/ethbridge/client/rest/rest.go

```diff
@@ -102,7 +104,7 @@ func createClaimHandler(cliCtx context.CLIContext) http.HandlerFunc {
 
 		// create the message
 		ethBridgeClaim := types.NewEthBridgeClaim(
-			req.EthereumChainID, bridgeContractAddress, req.Nonce, req.Symbol,
+			int64(req.EthereumChainID), bridgeContractAddress, int64(req.Nonce), req.Symbol,
```

Comment:
out of scope of this pr but we should size the nonce and id. Leaving it as int could potentially cause issues.

Reply:
Yeah, will add a todo in before I merge.